src/utils/createRecipeObject.js

```diff
@@ -37,6 +37,18 @@ const createRecipeObject = (
     notes: values.notes,
   }
 
+  if (ingredients && ingredients.data) {
+    ingredients.data.foreach(ingredient => {
+      if (ingredient.amount.toString().includes('/')) {
+        const slashPos = ingredient.amount.toString().search('/')
+        ingredient.amount = Number(
+          ingredient.amount.substring(0, slashPos) /
+            ingredient.amount.substring(slashPos + 1)
```

Comment:
You may be able to simplify this a little. you can do `const [a,b] = ingredient.amount.toString().split('/')` and that should give you `a` as the numerator and `b` as the denominator. Then the next line could just be `ingredient.amount = Number(a / b)`. Something along those lines may be a little more readable

Reply:
that's neat. Updating now

Reply:
added